association/classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students < student
+  end
```

Comment:
- [ ] Kindly implement the method that serves as the 'has-many' (a classroom has many students). like the foloowing example
![image](https://user-images.githubusercontent.com/81932478/192830153-421dc299-8961-45d6-874c-564e166668eb.png)